### PR TITLE
Add footway width quest

### PR DESCRIPTION
**Pull request by wielandb**
This PR adds a SCEE-specific quest that asks for the width of footways and the footway-parts of segregated foot/cycleways.

### CODE REVIEW

**Comment by Helium314** [comment]
Looks good, did you test it?

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -978,6 +978,7 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_cyclewayPartSurface_title">What’s the surface of the bicycle path here?</string>
 
     <string name="quest_cycleway_width_title">"How wide is the bicycle path here?"</string>
+    <string name="quest_footway_width_title">"How wide is the foot path here?"</string>
```
Comment (Helium314) [inline app/src/main/res/values/strings.xml:981]: Could you move the question text to the _new quests_  /_other_ section of `strings_ee.xml`?

Comment (wielandb) [reply app/src/main/res/values/strings.xml]: Done 👍

**Comment by wielandb** [comment]
> Looks good, did you test it?

I made sure the app builds and installed it on my phone. I solved the quest for a `width` and a `footway:width` situation and it added the expected tags. I hope this is an appropriate testing proccess.

**Pull request merged**